### PR TITLE
6.0.1 Release Notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
…a typo in the Config element guide.

### CODE REVIEW

File: docs/release_notes_128t_6.0.md [hunk 1]
```diff
@@ -24,6 +24,40 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+**Release Date:** August 15, 2022
+
+### New Features
+
+- **I95-46387 New Service Route Type for BGP over SVR:** Added a new service route type that indicates to the Routing Agent that local service routes and BGP over SVR routes should be merged together when creating local balanced paths.
+------
+- **I95-47077 Configuration options for User Accounts:** Added configuration options for number of login attempts before locking user account, and number of seconds that user account will be locked before being able to attempt to login again. For information, see [Password Policies](config_passworkd_policies.md).
+------
+- **I95-47136 Settings for WAN Link Speed and Duplex:** With this update, the SSR supports the auto-negotiation, disable, and speed/duplex settings configured in Mist on an SSR.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:37]: This is a Mist feature. The SSR already has this capability.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.0.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md]: Updated.

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart-jnpr — CHANGES_REQUESTED** [review]
Need the identified changes

File: docs/release_notes_128t_6.0.md [hunk 2]
```diff
@@ -24,6 +24,44 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+### Resolved Issues
+
+- **I95-38408 DHCP server on wrong vlan sends offer in response to discover message:** Hosted DHCP servers that do not have an explicit vlan configured are now explicitly treated as vlan 0, and handle any DHCP packets that are untagged/vlan 0, in order to prevent those packets from being multicasted to multiple DHCP servers.
+------
+- **I95-44434 Peer metric sends IP of WAN interface instead of the expected string:** Logic has been added to show the available destination address.
+------
+- **I95-46056 `show ntp` has no output from PCLI, even though NTP is configured:** The output of `show ntp` now reports IP addresses of the time servers rather than resolve hostnames.
+------
+- **I95-46281 Kernel Update:** Update to RHCK 8.6 for the latest security fixes.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md:47]: removed this

**Review by jeffmart-jnpr — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**